frame/vamm/src/tests.rs

```diff
@@ -69,6 +113,65 @@ prop_compose! {
 	}
 }
 
+prop_compose! {
+	fn timestamp()(
+		t in VammTimestamp::MIN..=VammTimestamp::MAX
+	) -> Option<VammTimestamp> {
+		Some(t)
+	}
+}
+
+prop_compose! {
+	fn get_vamm_state(config: TestVammState<Balance, VammTimestamp>)(
+		(base_asset_reserves, quote_asset_reserves, peg_multiplier) in min_max_reserve(),
+		closed in prop_oneof![timestamp(), Just(None)]
```

Comment:
If you want, you can push the `Some` to the `prop_compose!` below
```suggestion
		t in VammTimestamp::MIN..=VammTimestamp::MAX
	) -> VammTimestamp {
	    t
	}
}

prop_compose! {
	fn get_vamm_state(config: TestVammState<Balance, VammTimestamp>)(
		(base_asset_reserves, quote_asset_reserves, peg_multiplier) in min_max_reserve(),
		closed in prop_oneof![timestamp().prop_map(|t| Some(t)), Just(None)]
```